### PR TITLE
lndclient: Add chan event for ChannelEventUpdate_FULLY_RESOLVED_CHANNEL

### PR DESCRIPTION
**Pull request by yyforyongyu**
Depends on #83.

### CODE REVIEW

**Review by guggero — APPROVED** [review]
Can be rebased now. If the `v0.14.2-0` tag wasn't created yet, we can also wait until this is merged and then tag it. That way we don't need `v0.14.2-1` necessarily.

**Comment by yyforyongyu** [comment]
> Can be rebased now. If the `v0.14.2-0` tag wasn't created yet, we can also wait until this is merged and then tag it. That way we don't need `v0.14.2-1` necessarily.

Done!

**Review by carlaKC — APPROVED** [review]

**Pull request merged**
